frontend/src/index.css

```
@@ -232,4 +232,307 @@ h1 {
+    /* layered background: base gradient + subtle fiber/noise SVG + faint crease + stains */
+    background-image:
+      linear-gradient(180deg, var(--parchment-base-1) 0%, var(--parchment-base-2) 60%),
+      url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='400' height='400' viewBox='0 0 400 400'%3E%3Cfilter id='f'%3E%3CfeTurbulence baseFrequency='0.9' numOctaves='2' stitchTiles='stitch'/%3E%3CfeColorMatrix type='saturate' values='0'/%3E%3C/filter%3E%3Crect width='100%25' height='100%25' filter='url(%23f)' opacity='0.06'/%3E%3C/svg%3E"),
+      radial-gradient(1200px 200px at 10% 20%, rgba(0,0,0,0.02), transparent 8%),
+      radial-gradient(500px 200px at 90% 80%, rgba(0,0,0,0.02), transparent 8%);
+    background-blend-mode: normal, overlay, multiply, multiply;
+
+    box-shadow:
+      0 18px 36px rgba(0,0,0,0.18), /* soft ambient shadow */
+      inset 0 6px 18px rgba(255,255,255,0.06), /* subtle inner highlight */
+      inset 0 -6px 14px rgba(0,0,0,0.03);
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)